src/org/istio/testutils/Utilities.groovy

```diff
@@ -111,6 +111,16 @@ def sendNotification(notify_list) {
       sendToIndividuals       : true])
 }
 
+// init Testing Cluster.
+def initTestingCluster() {
+  def cluster = failIfNullOrEmpty(env.E2E_CLUSTER, 'E2E_CLUSTER is not set')
```

Comment:
LGTM functionally. 

General question: do tests have full authorization inside the cluster? Could someone submit a PR with code that modifies gcloud settings since this seems to grant full authorization inside the cluster?